schema.graphql

```diff
@@ -208,6 +208,65 @@ type NativeBalanceChange @entity {
   block: Block!
 }
 
+type PublicKey @jsonField {
+  type: String!
+  key: String!
+}
+
+type ValidatorDescription @jsonField(indexed: true) {
+  moniker: String!
+  identity: String
+  website: String
+  securityContract: String
+  details: String
+}
+
+type ValidatorCommission @jsonField {
+  # commission block in plain format
+  rate: String!
+  maxRate: String!
+  maxChangeRate: String!
+}
+
+type MsgCreateValidator @entity {
```

Comment:
Would you mind adding a comment that links to the relevant cosmos-sdk docs, please?

Reply:
do you think this link is enough? https://docs.cosmos.network/main/build/modules/staking#msgcreatevalidator